doc/sphinx-guides/source/admin/consents.rst

```diff
@@ -0,0 +1,108 @@
+Consents
+=========
+
+Dataverse offers a possibility of adding/editing consents that are for superusers only.
+If you are a logged-in superuser, you can access it by REST api.
```

Comment:
Without describing what consents are it is really hard to understand the rest of the text. (why should someone bother with creating/editing something when he doesn't know what is the purpouse of it)